LAD-AMA-Common/telegraf_utils/telegraf_config_handler.py

```diff
@@ -165,8 +165,8 @@ def parse_config(data, me_url, mdsd_url, is_lad, az_resource_id, subscription_id
                 if not is_vmi_rate_counter:
                     is_vmi_rate_counter = telegraf_json[omiclass][plugin][field]["displayName"] in vmi_rate_counters_list
             
-            if is_vmi_rate_counter:
-                min_interval = "1s"
+            # if is_vmi_rate_counter:
+            #     min_interval = "1s"
```

Comment:
Minor: If removal of this line helped to fix 1s vs. 60s interval for counters, it might mean that we used min_interval somewhere later in the code instead of using the provided value in config, so some subtle bug may still exist, although now it's not longer affected by our default min value.

Reply:
the min_interval value is initiated to a very large number in the code block above (line 161) and then later down in the for loop (line 228) when we iterate over every plugin, whenever we see an interval for a field lower than the current minimum we set it as the minimum. For ex, if cx sets CPU fields to- usage_active - 15sec, usage_system 10sec,  usage_user 30sec , we would go for min_interval of 10 sec polling to make sure we meet the minimum. This was our original design though it is certainly up for discussion/improvement.